test/e2e/tests/account/account-modal-account-type-change.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { strict as assert } from 'assert';
+import { Suite } from 'mocha';
+import { Anvil } from '@viem/anvil';
+import { Driver } from '../../webdriver/driver';
+import { DEFAULT_FIXTURE_ACCOUNT } from '../../constants';
+import FixtureBuilder from '../../fixture-builder';
+import { WINDOW_TITLES, withFixtures } from '../../helpers';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import Eip7702AndSendCalls from '../../page-objects/pages/confirmations/redesign/batch-confirmation';
+import ActivityListPage from '../../page-objects/pages/home/activity-list';
+import HomePage from '../../page-objects/pages/home/homepage';
+import { mockEip7702FeatureFlag } from '../confirmations/helpers';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import AccountDetailsModal from '../../page-objects/pages/dialog/account-details-modal';
+
+describe('Upgrade Account', function (this: Suite) {
+  it('an EOA account can be upgraded when triggering a batch tx from a dapp in an odd chain id', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
```

Comment:
I think we can remove this this, as we are not interacting with the test dapp in this test

Reply:
fixed 👍